config/jobs-async-upload/jobs-async-upload-s3-to-oci-template.yaml

```diff
@@ -35,6 +35,9 @@ parameters:
   - name: DESTINATION_CONNECTION
     description: "Name of the Connection resource for the Sink destination (e.g.: OCI)"
     required: true
+  - name: MODEL_SYNC_SOURCE_AWS_KEY
+    description: The path in the S3 bucket
+    required: true
```

Comment:
per ODH, we pass the KEY externally to the secret, hence this is a Parameter that we need as Env variable